pipettes/firmware/utility_gpio.c

```diff
@@ -77,6 +77,35 @@ void limit_switch_gpio_init() {
     }
 }
 
+void encoder_gpio_init() {
+//    PipetteType pipette_type = get_pipette_type();
```

Comment:
Why is this commented out?

Reply:
it was because it was being used for the encoder index pin initialization which we're no longer using- I'll take this out